pymetis/metis/CType.py

```diff
@@ -0,0 +1,2 @@
+RM = 0
```

Comment:
Doing these one enum per file makes no sense. Why not use classes to introduce namespaces?

Reply:
Thank you for your commands. This is a better idea than what I did. I will do it.